src/components/WalletUnlock/components/HDWallet.tsx

```diff
@@ -247,40 +236,64 @@ const DeterministicWallet = ({
           <Trans id="MNEMONIC_TITLE" />
         </Title>
       </HeadingWrapper>
-      <Typography>
-        <Trans id="MNEMONIC_SUBTITLE" />
-      </Typography>
-      <Parameters>
-        <AssetSelector
-          selectedAsset={assetToUse}
-          showAssetIcon={false}
-          showAssetName={true}
-          searchable={true}
-          assets={filteredAssets}
-          onSelect={(option: ExtendedAsset) => {
-            handleAssetUpdate(option);
-          }}
+
+      <Box pb={SPACING.MD} variant="spaceBetween">
+        <Box variant="columnAlignLeft" m={SPACING.XS}>
+          <Text>
+            <Trans id="MNEMONIC_SUBTITLE" />
+          </Text>
+          <AssetSelector
+            selectedAsset={assetToUse}
+            showAssetIcon={false}
```

Comment:
Shouldn't we display the icon?

Reply:
Just slows down the dropdown more. I'd say it'd be fine to add back the icon when we filter the assets list. But we don't have a way of separating important vs long-tail tokens, so it'd be better to avoid for now.